src/main.rs

```diff
@@ -1,14 +0,0 @@
-#![cfg_attr(feature = "strict", deny(warnings))]
```

Comment:
oh, I kept one locally, but I just `.gitignore`'d it - getting too specific / random to be worth checking in, imo